src/it/scala/org/zalando/spearheads/innkeeper/dao/RoutesPostgresRepoSpec.scala

```diff
@@ -6,10 +6,10 @@ import akka.actor.ActorSystem
 import akka.stream.ActorMaterializer
 import akka.stream.scaladsl.Source
 import org.scalatest.concurrent.ScalaFutures
-import org.scalatest.time.{ Seconds, Span }
-import org.scalatest.{ BeforeAndAfter, FunSpec, Matchers }
```

Comment:
after using scala style again, git doesn't show me any changes... I guess this was the correction...